OnlineShopping/CartService/WebApi/UI/Controllers/CartItemsController.cs

```diff
@@ -7,76 +7,75 @@
 using OnlineShopping.CartService.WebApi.DAL.Entities;
 using OnlineShopping.CartService.WebApi.UI.Controllers.Dtos;
 
-namespace OnlineShopping.CartService.WebApi.UI.Controllers
+namespace OnlineShopping.CartService.WebApi.UI.Controllers;
+
+[ApiController]
+[ApiVersion("1.0", Deprecated = true)]
+[ApiVersion("2.0")]
+[Route("api/v{version:apiVersion}/carts")]
+[Authorize(Policy = Policies.CRUD)]
+public class CartItemsController(
+        ICartItemsService _cartItemsService,
+        IMapper mapper) : ControllerBase
 {
-    [ApiController]
-    [ApiVersion("1.0", Deprecated = true)]
-    [ApiVersion("2.0")]
-    [Route("api/v{version:apiVersion}/carts")]
-    [Authorize(Policy = Policies.CRUD)]
-    public class CartItemsController(ILogger<CartItemsController> logger,
-            ICartItemsService _cartItemsService,
-            IMapper mapper) : ControllerBase
+    [HttpGet("{cartId}/items")]
+    [MapToApiVersion("1.0")]
+    [ApiExplorerSettings(GroupName = "v1")]
+    [ProducesResponseType<int>(StatusCodes.Status200OK)]
```

Comment:
- just a small idea, returning `ActionResult<int>` instead of `IActionResult` would remove the need to write the attribute here.
- 500 can be added by convention to all endpoints